python_modules/dagster/dagster/_core/definitions/run_config.py

```diff
@@ -448,6 +469,7 @@ def define_isolid_field(
     ignored: bool,
     is_using_graph_job_op_apis: bool,
     asset_layer: AssetLayer,
+    input_source_assets: Mapping[str, "SourceAsset"],
```

Comment:
would be nice to refactor this code to bunch of the "clumps" of parameters that are passed to every function a struct so a change like this isn't as noisy, but definitely don't need to change that for this PR